aptos-move/aptos-aggregator/src/types.rs

```diff
@@ -246,6 +246,17 @@ impl DelayedFieldValue {
             },
         })
     }
+
+    /// Approximate memory consumption of current DelayedFieldValue
+    pub fn get_approximate_memory_size(&self) -> usize {
```

Comment:
Why "approximate"?

Reply:
because I am not sure about the exact footprint of the vector, and actual size doesn't matter, as there is a lot of uncertainty in the maps that store these as well.